help/destinations/catalog/crm/hubspot.md

```diff
@@ -0,0 +1,188 @@
+
+### Authenticate to destination {#authenticate}
+
+Fill in the required fields below. Refer to the [Gather the [!DNL HubSpot] private app access token](#gather-credentials) section for any guidance.
+* **[!UICONTROL Bearer Token]**: The Access token for your [!DNL HubSpot] private app.
+
+To authenticate to the destination, select **[!UICONTROL Connect to destination]**.
+![Platform UI screenshot showing how to authenticate.](../../assets/catalog/crm/hubspot/authenticate-destination.png)
+
+If the details provided are valid, the UI displays a **[!UICONTROL Connected]** status with a green check mark. You can then proceed to the next step.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+![Platform UI screenshot showing the destination details.](../../assets/catalog/crm/hubspot/destination-details.png)
+
+* **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+* **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate audiences to this destination {#activate}
+
+>[!IMPORTANT]
+>
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and audiences to streaming audience export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audiences to this destination.
+
+### Map attributes and identities {#map}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL HubSpot] destination, you must go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination.
+
+To correctly map your XDM fields to the [!DNL HubSpot] destination fields, follow the steps below:
+
+* The identity, `Email` is a mandatory mapping, to set up follow the steps below:
```

Comment:
Line 125: please rephrase this to be clearer:

"The `Email` identity is a mandatory mapping for this destination. Follow the steps below to map it:"